zerver/lib/actions.py

```diff
@@ -5451,7 +5452,8 @@ def send_presence_changed(user_profile: UserProfile, presence: UserPresence) ->
         # stop sending them at all.
         return
 
-    presence_dict = presence.to_dict()
+    # The mobile app handles these events so we need to use the old format.
```

Comment:
We probably want to extend this comment to mention that we should make the `slim_presence` client capability control the format of these events once that is fully implemented.